package.json

```diff
@@ -78,15 +80,12 @@
     "prettier": "^2.8.8",
     "prettier-plugin-svelte": "^2.10.0",
     "prismjs": "^1.29.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
```

Comment:
Oh this isn't a required peer dep of storybook anymore?

Reply:
npm will auto-install peer deps, so it seemed ok to just let storybook handle it, since we don't care about React at all. Can revert this if you want!